tests/test_pyfileindex.py

```
@@ -413,3 +418,58 @@ def test_get_changes_quick(self):
             PyFileIndex(path="no_such_folder")
+
+
+class TestJobFileTableCoverage(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.path = os.path.dirname(os.path.abspath(__file__))
+        cls.fi = PyFileIndex(path=cls.path)
+
+    def test_scandir_import_error(self):
+        with patch.dict(sys.modules, {'os.scandir': None}):
+            import pyfileindex.pyfileindex
```

Ruff (0.12.2)

`import` should be at the top-level of a file

(PLC0415)